README.md

```diff
@@ -26,31 +26,51 @@ Next follow these instructions.
     PASSWORD: osp
     HOST: localhost
     ``` 
-    You may run the following commands for local setup of DB:
+    You may run the following commands for local setup of DB in Linux:
 
     ```
+    cd open-source-programs-backend
     sudo -i -u postgres
     createuser osp --pwprompt
     psql
     CREATE DATABASE osp;
     \c osp;
     GRANT ALL PRIVILEGES ON DATABASE osp to osp;
     ```
+2. you need to download API key file from your user-settings on Zulip. The file you download is named as 'download' or rename that to 'download'.
+3. Put that download file in the project's directory.
 
-
-2. To start the server:
+4. Move into the project's directory.
 
     ```
     cd open-source-programs-backend
+    ````
+5. You don't need to create virtual environment while working in Linux.
```

Comment:
You can though, I usually do this even on Linux. You can rephrase it to something like:
```
Create virtual environment (this is not a hard requirement, but its advisable)
```
This is so if the contributor has multiple projects with different dependencies, they can be in separate environments.